src/index.html

```diff
@@ -1,13 +1,36 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img class="card__image" src="images/imac.jpeg"
+      alt="comp">
+      <p class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      <p class="card__product">Product code: 195434</p>
+      <div class="stars stars--4">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <p class="reviews">Reviews: 5</p>
+      </div>
+
+
+      <div class="price">
+        <p class="price__text">Price:</p>
+        <p class="price__cost">$2,199</p>
+      </div>
+      <div class="button">
+        <a class="link" data-qa="hover"
+        href="/">BUY</a>
+      </div>
```

Comment:
```suggestion
    <div class="card" data-qa="card">
      <img 
        class="card__image" 
        src="images/imac.jpeg" 
        alt="comp"
      >
      <p class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
      <p class="card__product">Product code: 195434</p>
      <div class="card__stars stars stars--4">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <p class="card__reviews">Reviews: 5</p>
      </div>


      <div class="card__price">
        <p class="card__price-text">Price:</p>
        <p class="card__price-cost">$2,199</p>
      </div>
      
      <div class="card__button">
        <a 
          class="card__link" 
          data-qa="hover"
          href="/"
        >
          BUY
        </a>
      </div>
```
there should be only 2 BEM blocks, `card` and `stars`